samples/cellular/modem_shell/src/location/location_shell.c

```diff
@@ -444,7 +446,7 @@ int location_shell(const struct shell *shell, size_t argc, char **argv)
 
 	gnss_location_to_cloud = false;
 
-	while ((opt = getopt_long(argc, argv, "m:t:", long_options, &long_index)) != -1) {
+	while ((opt = getopt_long(argc, argv, "m:t:h", long_options, &long_index)) != -1) {
```

Comment:
`location get` doesn't give an error if there are parameters without a hyphen. For example `location get foobar` starts location retrieval. Would there be an easy way to check that and return an error?

Reply:
This is same with all commands using getopt and has always been. I haven't tried to add errors for all cases to keep things simple because the same error handling has to be copied into all places (currently 30 places in my working version). Something like this after all getopt_long while loops should do the trick:
```
	if (optind < argc) {
		mosh_error("Unknown argument '%s'", argv[optind]);
	}
```
Which would print as follows for `sms send -n 1234567890 ohhoh -m moikka jaaha`:
`Unknown argument 'ohhoh'`